registration/include/pcl/registration/impl/gicp.hpp

```diff
@@ -521,7 +522,8 @@ GeneralizedIterativeClosestPoint<PointSource, PointTarget>::computeTransformatio
                 getClassName().c_str(),
                 nr_iterations_,
                 max_iterations_,
-                (transformation_ - previous_transformation_).array().abs().sum());
+                static_cast<float>(
```

Comment:
Why cast to float here? `%f` can work with float and double

Reply:
I've removed the cast.